content/nginxaas-azure/getting-started/migrate-to-platform-metrics.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Migrate from Custom Metrics to Platform Metrics
+weight: 1000
+toc: true
+url: /nginxaas/azure/getting-started/migrate-to-platform-metrics/
```

Comment:
I'm wondering if "Getting Started" is the right section for this. Maybe this is better to have under [Logging and monitoring](https://docs.nginx.com/nginxaas/azure#) ?

Note: I have a similar concern with [Migrate from Standard to Standard V2](https://docs.nginx.com/nginxaas/azure/getting-started/migrate-from-standard/) but that is already where it is so we have to live with it until all Standard deployments have been migrated.